test/authUser/authRegister.js

```diff
@@ -0,0 +1,32 @@
+import { describe, it } from 'mocha';
+import { use, request, expect } from 'chai';
+import chaihttp from 'chai-http';
+import server from '../../index';
+
+use(chaihttp);
+
+describe('Post api/users', () => {
+  it('it register\'s a user', (done) => {
```

Comment:
Get rid of the escape string

Rephrase the line `'it register\'s a user'` to read `'it should register a user'`